projects/plugins/jetpack/modules/masterbar/admin-menu/admin-menu.js

```diff
@@ -111,28 +111,30 @@
 		}
 
 		if ( jetpackAdminMenu.isAtomic ) {
-			document.querySelectorAll( 'li.wp-has-submenu.wp-not-current-submenu' ).forEach( function ( el ) {
-				const submenu = el.querySelector( '.wp-submenu' );
-				const linkElement = el.querySelector( 'a' );
-
-				el.addEventListener( 'mouseover', function() {
-					submenu.style.display = null;
-					submenu.style.top = '-1px';
-					if ( ! isElementInViewport( submenu ) ) {
-						// Repoisition the submenu to the top of the menu item.
-						submenu.style.top = ( linkElement.clientHeight - submenu.clientHeight ) + 'px';
-					}
-					linkElement.focus();
-				} );
-
-				el.addEventListener( 'mouseleave', function() {
-					submenu.style.display = 'none';
-					submenu.style.top = null;
-					if ( document.activeElement === linkElement ) {
-						linkElement.blur();
-					}
+			document
+				.querySelectorAll( 'li.wp-has-submenu.wp-not-current-submenu' )
+				.forEach( function ( el ) {
+					const submenu = el.querySelector( '.wp-submenu' );
+					const linkElement = el.querySelector( 'a' );
+
+					el.addEventListener( 'mouseover', function () {
+						submenu.style.display = null;
+						submenu.style.top = '-1px';
+						if ( ! isElementInViewport( submenu ) ) {
+							// Repoisition the submenu to the top of the menu item.
+							submenu.style.top = linkElement.clientHeight - submenu.clientHeight + 'px';
+						}
+						linkElement.focus();
+					} );
+
+					el.addEventListener( 'mouseleave', function () {
+						submenu.style.display = 'none';
+						submenu.style.top = null;
+						if ( document.activeElement === linkElement ) {
+							linkElement.blur();
+						}
+					} );
```

Comment:
Is this supposed to be part of this PR?

Reply:
I think this has crept in from a trunk merge/rebase. Let's remove it!

Reply:
I removed it. It was because my formatter keeps formatting it.